OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/subscriptions/impl/SubscriptionManager.kt

```diff
@@ -157,7 +157,10 @@ internal class SubscriptionManager(
         args: ModelChangedArgs,
         tag: String,
     ) {
-        val subscription = subscriptions.collection.firstOrNull { it.id == args.model.id }
```

Comment:
I see, to use `id` as a getter, it would have to compare `it.model.id == args.model.id`

Reply:
Ya, since the [Susbcription.id](https://github.com/OneSignal/OneSignal-Android-SDK/blob/5.1.6/OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/Subscription.kt#L14) returns `""` if it starts with `local-` it was not finding it